esrally/racecontrol.py

```diff
@@ -288,10 +288,10 @@ def race(cfg, sources=False, build=False, distribution=False, external=False, do
 
 def set_default_hosts(cfg, host="127.0.0.1", port=9200):
     configured_hosts = cfg.opts("client", "hosts")
-    if len(configured_hosts()) !=0 :
-        logger.info("Using configured hosts %s" % configured_hosts())
+    if len(configured_hosts.default) !=0 :
+        logger.info("Using configured hosts {}".format(configured_hosts.default))
     else:
-        logger.info("Setting default host to [%s:%d]" % (host, port))
+        logger.info("Setting default host to [{}:{}]".format(host, port))
```

Comment:
I think we settled on using the idiom `logger.info("...%s...", parameter0, parameter1)`?

Reply:
Ugh too fast replacing things here. Will revert.

Reply:
Addressed in 0a17876dd048e0720d5104f6d5897d31de5dc94f